actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add support to `ActionDispatch::SystemTestCase::driven_by` for specifying the Capybara driver name in order to allow users to prevent conflicts among drivers that use the same driver type (selenium, poltergeist, webkit, rack test).
```

Comment:
Knit: could you wrap the CHANGELOG to 80 chars, I know this isn't always followed (like the log below 👇 ) but we do [ask that you wrap them "around 80 chars"](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog). 🙏

Reply:
Fixed in 3611dd2d8e